### ISSUE TITLE
"Error rebuilding zip file [[WinError 32]" when tagging a CBZ with existing ComicInfo.xml

### ISSUE THREAD

**Issue by nosoop4u** [Metron-Project/metron-tagger#145]
Every time I scrape a file (`metron-tagger -o <filename>`) that already has been populated with metadata, I get the following (_XXXXXXX_ is just a series of random characters in the output below):

```
Starting online search and tagging:
----------------------------------
Error rebuilding zip file [[WinError 32] The process cannot access the file because it is being used by another process: 'C:\\temp\\tmpXXXXXXX']: filename.cbz
```

The result is I have 2 new files: 
1. _filename_ (the CBZ file passed as a parameter) - file appears to be a valid ZIP containing just the new ComicInfo.xml, but it is around the same size as the original file.  
2. tmpXXXXXXXX - file in the same directory, which also appears to be the original CBZ file passed to metron-tagger, but with the original ComicInfo.xml (prior to running metron-tagger) removed.

This is on Windows 10 22H2 running on a Intel i7-6700HQ with 16GB RAM, with python 3.9.13, and metron-tagger 1.6.4.

I tried this with a half dozen different CBZ files that already had ComicInfo.xml files and the behavior was the same, whether the ComicInfo.xml was created by ComicRack or metron-tagger.

There doesn't appear to be any other process locking the file beyond metron-tagger itself because I can modify it right after.

If I delete the ComicInfo.xml from the CBZ before scraping with metron-tagger, then this error does not occur!

Another note, I installed metron-tagger in a Linux LXC under Proxmox, and it does not have this issue in that environment.

**To Reproduce** Steps to reproduce the behavior:

Run metron-tagger -o _filename_ in an environment like the above (Win 10, Python 3.9) using any CBZ with a pre-existing ComicInfo.xml

**Expected behavior** A clear and concise description of what you expected to
happen.

Perhaps this is a timing issue, and a delay needs to be inserted when modifying the ZIP/CBZ?   I would hope the result of the above command would be an updated CBZ file with a new ComicInfo.xml with the data scraped from Metron.

Note I get the same error when running metron-tagger whether the underlying storage holding the comic file is a regular HDD or a fast NVMe.

**Screenshots** If applicable, add screenshots to help explain your problem.

**Environment details** 

Provided in description above

**Additional context**
NOTE: The metron-tagger was installed using:
 
`pip install --trusted-host=pypi.python.org --trusted-host=pypi.org --trusted-host=files.pythonhosted.org --upgrade metron-tagger`

because the install command in github yielded the following error:

`ERROR: Could not install packages due to an EnvironmentError: Could not find a suitable TLS CA certificate bundle, invalid path: /etc/pki/tls/certs/ca-bundle.crt`

Thanks for making this program and the metadata website.  Great stuff!

**Comment by bpepple** [comment]
You are using a pretty old version MT, and if I remember correctly this was a bug that affected only Windows. If you update to the latest version of MT this should be fixed (not to mention removal of files from the Zipfile are significantly faster).

**Comment by nosoop4u** [comment]
I installed it using the method outlined on this github page(pip install), and that was the version installed.  Is there a different command I should be using to get the latest version?

**Comment by nosoop4u** [comment]
```
$ pip index versions metron-tagger
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
metron-tagger (1.6.4)
Available versions: 1.6.4, 1.6.3, 1.6.2, 1.6.1, 1.6.0, 1.5.0, 1.4.4, 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.1, 1.3.0, 1.2.18, 1.2.16, 1.2.14, 1.2.12, 1.2.10, 1.2.9, 1.2.8, 1.2.7, 1.2.6, 1.2.4, 1.2.2, 1.2.1, 1.2.0, 1.1.6, 1.1.4, 1.1.2, 1.1.0, 1.0.18, 1.0.16, 1.0.15, 1.0.14, 1.0.12, 1.0.10, 1.0.9, 1.0.7, 1.0.6, 1.0.5, 1.0.2, 1.0.1, 1.0.0, 0.9.9, 0.9.5, 0.9.0, 0.8.0, 0.7.0, 0.6.0, 0.5.0, 0.1.0
  INSTALLED: 1.6.4
  LATEST:    1.6.4
```

**Comment by nosoop4u** [comment]
OK, I updated to a more recent python (3.12) and was then able to install latest version of metron-tagger.
Must be dependencies.
Leaving this here in case it helps anyone else.
